restart_process/README.md

```diff
@@ -60,11 +60,15 @@ def docker_build_with_restart(ref: str, context: str,
 ```
 ## Unsupported Cases
 This extension does NOT support process restarts for:
-- Images run in Docker Compose resources
+- Images run in Docker Compose resources (use the [`restart_container()`](https://docs.tilt.dev/api.html#api.restart_container) builtin instead)
 - `custom_build`
 - Images without a shell (e.g. `scratch`, `distroless`)
+- Container commands specified as `command` in Kubernetes YAML will be overridden by this extension.
+  - However, the `args` field is still available; [reach out](https://tilt.dev/contact) if you need help navigating the interplay between Tilt and these YAML values
```

Comment:
this feels like it's worth mentioning otherwise ppl might be surprised, but i'm not sure this comment is more clarifying than confusing? thoughts?

Reply:
eh, I think it's fine